spec/refile/spec_helper.rb

```diff
@@ -38,7 +38,8 @@ def presign
   File.open(tempfile.path, "r")
 end
 
-Refile.processor(:convert_case) do |file, format:|
+Refile.processor(:convert_case) do |file, options = {}|
+  format = options.fetch :format
```

Comment:
This actually seems to be a bug in JRuby. Filed: https://github.com/jruby/jruby/issues/2518